spec/services/support_interface/notification_preferences_export_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::NotificationPreferencesExport do
+  let(:provider1) { create(:provider) }
+  let(:provider2) { create(:provider) }
+  let(:provider_user1) { create(:provider_user, providers: [provider1, provider2]) }
+  let(:provider_user2) { create(:provider_user, providers: [provider2]) }
+
+  before do
+    provider_user1.provider_permissions.where(provider: provider2).update(make_decisions: true)
+    create(:provider_user_notification_preferences, provider_user: provider_user1)
+    provider_user2.provider_permissions.where(provider: provider2).update(make_decisions: true)
+    create(:provider_user_notification_preferences, provider_user: provider_user2)
+
+    create(:provider_user_notification_preferences_audit, notification_preferences: provider_user1.notification_preferences, changes: {
+      'application_received' => [true, false], 'offer_accepted' => [false, true], 'offer_declined' => [true, false]
+    })
+    create(:provider_user_notification_preferences_audit, notification_preferences: provider_user2.notification_preferences, changes: {
+      'application_withdrawn' => [true, false], 'application_rejected_by_default' => [false, true], 'offer_accepted' => [true, false]
+    })
+    create(:provider_user_notification_preferences_audit, notification_preferences: provider_user2.notification_preferences, changes: {
+      'application_received' => [true, false], 'offer_accepted' => [true, false]
+    })
+  end
+
+  describe '#data_for_export' do
```

Comment:
can you also add `it_behaves_like 'a data export'` to check for the docs?

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4905/commits/4b08a81937c773ce0b0c981ca24d6f6c3fe9291a#diff-3d837e8df52a2a6b66540ed205d8dd04d92fa9032dfeb58b4d4cbadcf200ef1dR26